Library/Formula/voikko-fi.rb

```diff
@@ -0,0 +1,26 @@
+class VoikkoFi < Formula
```

Comment:
The description of `resource` from formula cookbook looks like it's meant for resources that don't have a separate build step like voikko-fi. Voikko-fi is built from source files using foma, which isn't required for end use.